### DISCUSSION TITLE
Docker build failing

### DISCUSSION THREAD

**Discussion by bennet-esyoil** [postalserver/postal#1593] [Development discussions]
Hello,

docker build is currently failing for me. I did not change anything in the docker-file itself, so I guess there are some dependencys that broke the build.

See error log here: https://github.com/bennet-esyoil/postal/runs/3608497129?check_suite_focus=true

Best Regards

**Comment by bennet-esyoil** [comment]
```docker
# Install main dependencies
RUN apt-get update && \
  apt-get install -y --no-install-recommends \
  build-essential  \
  netcat \
  curl \
  libmariadb-dev \
  libcap2-bin \
  nano \
  nodejs
```

replacing this in the Dockerfile works for me.

**Reply by willpower232** [reply]
Interesting, I wonder if `--no-install-recommends` was innocently causing that and now the list needs to be longer

**Reply by bennet-esyoil** [reply]
iirc throwing out the `--no-install-recommends` wasn't fixing the issue. Was my first guess too.

But the `libcap2-bin` was completely missing and `libmariadbclient-dev` is no longer a package, just `libmariadb-dev` should be working fine.

**Reply by bennet-esyoil** [reply]
I opened a PR where this fix is included #1594
